packages/core/ts/MobileCore.ts

```diff
@@ -160,17 +145,6 @@ const MobileCore: IMobileCore = {
     return RCTAEPCore.getSdkIdentities();
   },
 
-  /**
-   * Called by the extension public API to dispatch an event for other extensions or the internal SDK to consume.
-   * Any events dispatched by this call will not be processed until after `start` has been called.
-   *
-   * @param event Required parameter with {@link Event} instance to be dispatched. Should not be nil
-   * @return true if the the event dispatching operation succeeded, otherwise the promise will return an error
-   */
-  dispatchEvent(event: Event): Promise<boolean> {
```

Comment:
Should we still keep this API? 
Android: https://github.com/adobe/aepsdk-core-android/blob/257d03653e341d26bf82e597dc7b01b705503c73/code/core/src/phone/java/com/adobe/marketing/mobile/MobileCore.java#L266
iOS: https://github.com/adobe/aepsdk-core-ios/blob/40a25aec18cacaa95b5d34a35ec5318feb926871/AEPCore/Sources/core/MobileCore.swift#L115